### ISSUE TITLE
A simple powershell script

### ISSUE THREAD

**Issue by GeopJr** [henrypp/chrlauncher#102]
I don't know if it's supposed to or not , but when chrlauncher updates ungoogled chromium , it gets installed on a folder , and not on /bin/ , which is , for me at least , annoying to manually move everytime.
So I created this [script](https://github.com/GeopJr/GeopJr.github.io/blob/master/update.ps1)
# BACKUP BEFORE USING FOR THE FIRST TIME
## Place it in your chrlauncher\64 or chrlauncher\32 folder
After placing it , you can create a shortcut to your desktop for easy access
This is the [script](https://github.com/GeopJr/GeopJr.github.io/blob/master/update.ps1)
```
cd bin
Move-Item ungoogled-chromium* ../
cd ../
Remove-Item .\bin -Force -Recurse
Move-Item ungoogled-chromium* bin
```
it will move the ungoogled-chromium folder that got downloaded and "installed"
to your /64 or /32 folder , then it will remove the /bin folder
and then it will rename the ungoogled-chromium folder to bin

# I AM NOT RESPONSIBLE FOR DELETED FILES
# IF THE AUTHOR BELIEVES THIS SHOULD BE DELETED (from here and from my repo), I WILL DELETE IT

**Comment by woolyss** [comment]
Hello, Thank you. That's interesting!

**Comment by biberesser** [comment]
This seems to explain the broken behavior when starting fresh with "ungoogled-chromium":

"ungoogled-chromium" is being installed to:
`%appdata%\Local\Chromium\bin\ungoogled-chromium_67.0.3396.87-3_windows`

steps to reproduce (fresh chrlauncher install*):
- installer always downloads the setup
- installer installs into "wrong" directory (see above)
- after installing, nothing happens
- when restarting chrlauncher, the procedure repeats (download, install, nothing happens)


* remark:
I installed chrlauncher (portable) via Chocolatey package manager:
https://chocolatey.org/packages/chrlauncher.portable
`choco install chrlauncher.portable -params '"/Type:ungoogled-chromium" /KeepSettings'`

**Comment by woolyss** [comment]
Now, the path issue is corrected in the new chrlauncher 2.5+ versions.
Thanks a lot Henry! ;)
